src/cmd/completion.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var completionNoDesc bool
+
+var completionCmd = &cobra.Command{
+	Use:   "completion [bash|zsh|fish|powershell]",
```

Comment:
The convention we have been using is `<` for required arguments and `[` for optional arguments.

```suggestion
	Use:   "completion <bash|zsh|fish|powershell>",
```